index.rst

```diff
@@ -48,7 +48,353 @@
+
+RSP Portal Aspect
+-----------------
+
+As in DP0.2 and as as planned for ops, the Portal Aspect obtains data through a combination
+of direct access to IVOA-standard interfaces, mostly importantly TAP queries and DataLink links services,
+and through DataLink-compatible but not specifically standardized data services discovered via DataLink.
+
+Image Data
+^^^^^^^^^^
+
+The Portal for PP Preview will access image data from TAP queries to ObsCore-formatted tables.
+
+The equivalent of data-rights-user access to post-embargo data will be via a true "ObsTAP" service,
+with the table named ``ivoa.ObsCore``.
+That table will represent the image data in the post-embargo Butler repository.
+
+Early access to pre-release data will be via tables representing ``/repo/embargo`` or any other Butler
+repositories that are relevant.
+Users will be able to select these tables in the PP Preview Portal's TAP screen, and when they do so,
+as is already true for the prototype ``live ObsCore`` service, the Portal will display the same
+search screen that it does for true "ObsTAP" services.
+
+If and when CAOM2 metadata becomes available, it will be searchable via TAP in the Portal Aspect, but the
+PP Preview era Portal will not provide any dedicated CAOM2-based search capabilities.
+
+Catalog Data
+^^^^^^^^^^^^
+
+As noted above, toward the end of PP Preview the DIA catalog products will be replicated to the Postgres PPDB.
+Portal access to that data will simply be via TAP in a way that has largely been exercised in DP0.2 and DP0.3.
+
+As notec abov, what is new here is the evolving nature of the ``DiaObject`` table, in particular.
+While that's not *per se* a problem for the Portal Aspect, we know that there are both user expectations
+and actual requirements (*vide supra*) on the reproducibility of queries.
+
+(TBS: list of requirements)
+
+PP Preview will allow these use cases to be exercised in the Portal Aspect, and will help us to
+determine whether we need a dedicated UI capability to support capturing PPDB queries in a
+reproducible form.
+
+RSP Notebook Aspect
+-------------------
+
+There are no special considerations for the Notebook Aspect beyond the above,
+but we summarize here the implications for Notebook users and highlight the difference
+between using the PP Preview environment simply for the staff's own data access needs,
+as opposed to a rehearsal of ops-era science-user data access.
+
+The principal path to image data access, in PP Preview as well as in operations,
+will be via access to a Butler.
+
+As in DP0.2, it will also be possible to invoke the IVOA image services from a Notebook,
+with the same mechanism we now have for making it easy for users to supply appropriate
+authentication tokens to those service invocations.
+This may be the most convenient path for users to invoke some of the image-adjacent
+services that will be advertised via the "links service" in our image data service
+model.
```

Comment:
Please add a link to reference material for how a user would invoke the IVOA image services from a notebook.